lib/Firtool/Firtool.cpp

```diff
@@ -25,6 +25,10 @@ using namespace circt;
 
 LogicalResult firtool::populatePreprocessTransforms(mlir::PassManager &pm,
                                                     const FirtoolOptions &opt) {
+
+  pm.nest<firrtl::CircuitOp>().addNestedPass<firrtl::FModuleOp>(
+      firrtl::createCheckInitPass());
```

Comment:
Consider enabling this separate from introducing it?

What's the plan for ExpandWhens once this lands (at least its initialization-checking features)?

Reply:
ExpandWhens no longer would need to check this and the code can be removed.